src/api/handlers/webhook.ts

```diff
@@ -227,33 +243,59 @@ export function getSlashCommands(guildID?: string) {
 /**
  * Edit an existing slash command. If this command did not exist, it will create it.
  */
-export function upsertSlashCommand(options: UpsertSlashCommandOptions) {
-  return RequestManager.post(
-    options.guildID
-      ? endpoints.COMMANDS_GUILD_ID(botID, options.id, options.guildID)
-      : endpoints.COMMANDS_ID(botID, options.id),
-    {
-      ...options,
-    },
-  );
+export async function upsertSlashCommand(
+  commandID: string,
+  options: CreateApplicationCommandOptions,
+  guildID?: string,
```

Comment:
Move `commandID` & `guildID` to `options`

Reply:
Same as above